traffic_ops/install/bin/postinstall

```diff
@@ -25,7 +28,7 @@ done
 
 # above options don't require any of the extra processing -- just kick off the script with
 # options intact.
-if [[ $action == "bypass" ]]; then
+if [[ -v action && "$action" == "bypass" ]]; then
```

Comment:
What does `-v` do here?

Reply:
`-v` checks if `action` is defined without expanding it. It's generally used in combination with `set -o nounset` (`set -u`).

Reply:
ah, thanks, I couldn't find it in the ancient version of bash that macOS includes by default